src/Navigation/SidePanel/__snapshots__/SidePanel.spec.tsx.snap

```diff
@@ -0,0 +1,189 @@
+  top: 10%;
+  left: 300px;
+  cursor: pointer;
+}
+
+.c3 {
+  transition: all 0.3s ease-out;
+}
+
+.c3:hover {
+  opacity: 0.7;
+}
+
+.c0 {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+  padding: 5px 25px;
+  width: 300px;
+  height: 0px;
+  left: -NaNpx;
+  top: 0px;
+  box-shadow: 0px 4px 5px rgba(0, 0, 0, 0.1);
+  transition: all 0.7s ease-in-out;
+}
+
+.c4 {
+  background: inherit;
+  border: 0;
+  font: inherit;
+  text-align: left;
+  cursor: pointer;
+  position: relative;
+  width: 100%;
+  height: 60px;
+  padding-left: calc(1em * 41 / (16 * 1.2));
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.5em;
+  border-top-left-radius: 0.75em;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0.75em;
+}
+
+.c4 >* {
+  display: block;
+  padding: 0;
+  margin: 0;
+  width: 90%;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  cursor: pointer;
+}
+
+.c4 >label {
+  font-size: 1.2em;
+  font-weight: bolder;
+}
+
+.c4 >span {
+  font-size: 1em;
+}
+
+.c4::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: unset;
+  width: 1em;
+  background: #e0e0e0;
+}
+
+.c4:focus-visible {
+  outline: 1px #e0e0e0 solid;
+}
+
+<div
+  className="c0"
+  onAnimationEnd={[Function]}
+  style={
+    {
+      "isOpen": false,
+      "width": undefined,
+    }
+  }
+>
+  <div
+    className="c1"
+    onClick={[Function]}
+    style={
+      {
+        "isOpen": false,
+        "width": undefined,
+      }
+    }
+  >
+    <svg
+      fill="none"
+      height="20"
+      viewBox="0 0 22 25"
+      width="20"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M0.499998 0L10.5 -1.74845e-06C16.8513 -2.85895e-06 22 5.14872 22 11.5L22 13.4677C22 19.819 16.8513 24.9677 10.5 24.9677L0.500002 24.9677L0.499998 0Z"
+        fill="#686867"
+        opacity="0.8"
+      />
+      <path
+        d="M7.0193 7.4903L11.9435 11.4297C12.6191 11.9701 12.6191 12.9976 11.9435 13.538L7.0193 17.4774"
+        stroke="white"
+        strokeLinecap="round"
+        strokeWidth="0.9"
+      />
+    </svg>
+  </div>
+  <div
+    className="c2"
+  >
+    test heading (undefined)
+  </div>
+  <div
+    className="c3"
+  >
+    <button
+      background="inherit"
+      className="c4"
+      flashColor="#e0e0e0"
+      height="60px"
+      id="id1"
+      onClick={[Function]}
+      orientation="left"
+      width="100%"
+    >
+      <label
+        htmlFor="id1"
+      >
+        test no sub
+      </label>
+    </button>
+  </div>
+  <div
+    className="c3"
+  >
+    <button
+      background="inherit"
+      className="c4"
+      flashColor="#e0e0e0"
+      height="60px"
+      id="id2"
+      onClick={[Function]}
+      orientation="left"
+      width="100%"
+    >
+      <label
+        htmlFor="id2"
+      >
+        test-tittle suite
```

Comment:
Maybe change **tittle** from `test-tittle` as this means something else ?

Reply:
it's a test this is fine. Let's get this merged please